src/libraries/System.Memory.Data/ref/System.Memory.Data.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
Is the _net461_ TFM needed here? _netstandard2.0_ should be a proper subset.

Reply:
It lets the net461 ref unify with the GAC through all of the facades, which reduces the xcopy size of an application that uses the package.  All of our OOBs do this now.

Reply:
Everything that we build for `netstandard2.0` we also build for `net461` see https://github.com/dotnet/runtime/issues/1625